tests/kernel_bundle/get_kernel_bundle_without_kernel_attr.cpp

```diff
@@ -0,0 +1,79 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  This test calls these sycl::get_kernel_bundle overloads:
+//    1) sycl::get_kernel_bundle<BundleState>(context, devices, selector)
+//    2) sycl::get_kernel_bundle<BundleState>(context, selector)
+//    3) sycl::get_kernel_bundle<BundleState>(context, devices)
+//    4) sycl::get_kernel_bundle<BundleState>(context, kernelIds)
+//    5) sycl::get_kernel_bundle<BundleState>(context, devices, kernelIds)
+//    6) sycl::get_kernel_bundle<BundleState>(context)
+//    7) sycl::get_kernel_bundle<KernelName, BundleState>(context)
+//    8) sycl::get_kernel_bundle<KernelName, BundleState>(context, devices)
+//  For kernels that uses fp16 and fp64 datatypes, kernel that uses atomic_ref
+//  (it requires atomic64 aspect) and for two kernels without any requirements
+//  for device from chosen platform.
+//
+//  The test verifies that kernel bundle that obtained from each
+//  get_kernel_bundle overload contain only kernels that compatible with
+//  provided device.
```

Comment:
```suggestion
//  The test verifies that the kernel bundle which is  obtained from each
//  get_kernel_bundle overload contains only kernels compatible with
//  the provided device.
```

Reply:
this comment was applied [here](https://github.com/KhronosGroup/SYCL-CTS/pull/224/commits/c151a8cdae55ff2813aa9db1ffde34b33ebd1c11#diff-cfce662345b154bd62edfd6bfceb4dc9e924282661c3234f57122c0262abe876R20)